fades/main.py

```diff
@@ -100,8 +106,9 @@ def go():
     parser.add_argument('-d', '--dependency', action='append',
                         help="specify dependencies through command line (this option can be "
                              "used multiple times)")
-    parser.add_argument('-r', '--requirement',
-                        help="indicate from which file read the dependencies")
+    parser.add_argument('-r', '--requirement', action='append',
+                        help="indicate files to read dependencies from (this option can be "
```

Comment:
Thanks for changing the help string here. But remember also to change the README and the man page to reflect this change.